src/backend/db/lib/seed.test.ts

```diff
@@ -0,0 +1,22 @@
+import test from "ava";
+import { getTestDatabase } from "@/backend/tests/fixtures/get-test-database";
+import { getDb } from "@/backend/db/lib/get-db";
+import { seedfile } from "@/backend/db/lib/seed";
+
+test("seedFile works after schema changes", async (t) => {
```

Comment:
Minor suggestion: I think "seedFile works with current schema" would be a slightly better title. The "after schema changes" part could imply that the test itself is applying schema changes before running seedFile.

Reply:
great callout @mrabbitt ! ... changed in 2c0415ac4d099d6c4602222a2dbd7ac660c91d9e to 'seedfile works with current schema'